docs/architecture/cloud-native/toc.yml

```diff
@@ -43,14 +41,14 @@
       href: front-end-communication.md
     - name: Cross-Service Communication
       href: cross-service-communication.md
-  - name: Distributed Data
+  - name: Cloud-Native Data Patterns
     href: distributed-data.md
```

Comment:
If the title is being renamed, then in this case the file should renamed to data-patterns.md